metadata_tools/hosts/GO_0xxx/gcp_index_startup.sh

```
@@ -0,0 +1,22 @@
+
+
+apt-get install -y python3 python3-pip python3-venv git
+cd
+git clone https://github.com/SETI/rms-metadata-tools.git
+cd rms-metadata-tools
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+export MY_WORKER_DEST_BUCKET=gs://my-bucket/results
+#python3 GO_0xxx_index_cloud.py gs://rms-node-holdings/pds3-holdings/volumes/GO_0xxx/ gs://rms-node-metadata/GO_0xxx/ --index_tree gs://rms-node-holdings/pds3-holdings/metadata/GO_0xxx/
+cd hosts/GO_0xxx
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)